### PR TITLE
Use an image from a URL instead of uploading

### PR DESCRIPTION
**Pull request by gpanders**
For now the "Use an image from the web" icon is the same as the "Upload image" icon since I cannot modify the sprites psd file. I leave that to the author. But the rest of the code for this feature is in this PR.

### CODE REVIEW

File: templates/t_itemEdit.mustache [hunk 1]
```diff
@@ -4,21 +4,34 @@
     </span>
     <span class="lpImageCell">
         {{#image}}
+            <!-- <img class="lpItemImage" src="{{imageUrl}}" /> -->
             <img class="lpItemImage" src="https://i.imgur.com/{{image}}s.jpg" />
         {{/image}}
+        {{^image}}
+          {{#imageUrl}}
```
Comment (galenmaly) [inline templates/t_itemEdit.mustache]: [nit] indent 4 spaces please

File: templates/t_itemEdit.mustache [hunk 2]
```diff
@@ -4,21 +4,34 @@
     </span>
     <span class="lpImageCell">
         {{#image}}
+            <!-- <img class="lpItemImage" src="{{imageUrl}}" /> -->
```
Comment (galenmaly) [inline templates/t_itemEdit.mustache]: If this is no longer needed, can remove

File: app.js [hunk 3]
```diff
@@ -266,6 +266,7 @@ app.post("/register", function(req, res) {
 
     db.users.find({username: username}, function(err, users) {
         if( err || users.length) {
+            awesomeLog(req, "User Exists.");
```
Comment (galenmaly) [inline app.js]: Thanks for adding these

**Comment by gpanders** [comment]
Newest commit combines the two options into a single modal. Reverted CSS changes from original commit that modified the actions bar

**Comment by galenmaly** [comment]
Looks great, thanks!

**Comment by galenmaly** [comment]
This is now deployed. I made some UI tweaks before deploying: https://github.com/galenmaly/lighterpack/commit/b29a4ce7d8aa0715579a13c2f9bbb67e191b5721

**Comment by galenmaly** [comment]
One enhancement that would be good to see would be the ability to switch between which image the user wants to use. Right now once an image has been uploaded, it takes over and the user can't use imageUrl again.

**Pull request merged**